clusters/build-clusters/01_cluster/openshift/ci-ns-ttl-controller/ci-ns-ttl-controller_dc.yaml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
+metadata:
+  labels:
+    app: namespace-ttl-controller
+  name: namespace-ttl-controller
+  namespace: ci
+spec:
+  replicas: 1
+  revisionHistoryLimit: 2
+  selector:
+    app: namespace-ttl-controller
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: namespace-ttl-controller
+    spec:
+      serviceAccount: namespace-ttl-controller
+      containers:
+      - name: namespace-ttl-controller
+        image: ci-ns-ttl-controller:latest
```

Comment:
@stevekuznetsov 

Saw [you comment](https://jira.coreos.com/browse/DPTP-647?focusedCommentId=144384&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-144384) in the jira card

We could use the full path here and remove the trigger of imagestream as we do for k8s deployment.
And also remove the imagestream sync below.
But the image is always the same as `ci-ns-ttl-controller:latest` (which is the reason that upstream uses sha1 in the tag???).

So it would not trigger new deployment when our `applyconfig` applys things here, right?
Maybe this is the reason you use `dc` in the first place?